test/new-e2e/tests/npm/ec2_1host_wkit_test.go

```diff
@@ -35,8 +34,8 @@ func TestEC2VMWKitSuite(t *testing.T) {
 func (v *ec2VMWKitSuite) SetupSuite() {
 	v.BaseSuite.SetupSuite()
 
-	v.Env().RemoteHost.MustExecute("Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))")
-	v.Env().RemoteHost.MustExecute("C:\\ProgramData\\chocolatey\\choco install /y apache-httpd --params '\"/noService\"'")
+	v.Env().RemoteHost.MustExecute("Invoke-WebRequest -UseBasicParsing http://s3.amazonaws.com/dd-agent-mstesting/windows/pvt/nplanel/httpd-2.4.59-240404-win64-VS17.zip -OutFile httpd.zip")
```

Comment:
Is this bucket/object stable enough to put in a test?  

How does one update the contents?

Reply:
I use our microsoft testing bucket describe [here](https://datadoghq.atlassian.net/wiki/spaces/BARX/pages/1611595838/S3+buckets) for the moment
I will move it to stable once I have a path and would be tested for multiple weeks

BTW That the recommended solution by windows folks